modules/gdnative/godot/godot_variant.cpp

```diff
@@ -53,461 +51,432 @@ void GDAPI godot_variant_copy(godot_variant *p_dest, const godot_variant *p_src)
-	const Variant *v = (const Variant *)p_v;
-	godot_input_event ev;
-	InputEvent *event = (InputEvent *)&ev;
-	*event = *v;
-	return ev;
-}
-
-godot_dictionary GDAPI godot_variant_as_dictionary(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_dictionary dict;
-	godot_dictionary_new(&dict);
-	Dictionary *d = (Dictionary *)&dict;
-	*d = *v;
-	return dict;
-}
-
-godot_array GDAPI godot_variant_as_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_array array;
-	godot_array_new(&array);
-	Array *a = (Array *)&array;
-	*a = *v;
-	return array;
-}
-
-godot_pool_byte_array GDAPI godot_variant_as_pool_byte_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_byte_array pba;
-	godot_pool_byte_array_new(&pba);
-	PoolByteArray *p = (PoolByteArray *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_pool_int_array GDAPI godot_variant_as_pool_int_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_int_array pba;
-	godot_pool_int_array_new(&pba);
-	PoolIntArray *p = (PoolIntArray *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_pool_real_array GDAPI godot_variant_as_pool_real_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_real_array pba;
-	godot_pool_real_array_new(&pba);
-	PoolRealArray *p = (PoolRealArray *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_pool_string_array GDAPI godot_variant_as_pool_string_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_string_array pba;
-	godot_pool_string_array_new(&pba);
-	PoolStringArray *p = (PoolStringArray *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_pool_vector2_array GDAPI godot_variant_as_pool_vector2_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_vector2_array pba;
-	godot_pool_vector2_array_new(&pba);
-	PoolVector2Array *p = (PoolVector2Array *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_pool_vector3_array GDAPI godot_variant_as_pool_vector3_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_vector3_array pba;
-	godot_pool_vector3_array_new(&pba);
-	PoolVector3Array *p = (PoolVector3Array *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_pool_color_array GDAPI godot_variant_as_pool_color_array(const godot_variant *p_v) {
-	const Variant *v = (const Variant *)p_v;
-	godot_pool_color_array pba;
-	godot_pool_color_array_new(&pba);
-	PoolColorArray *p = (PoolColorArray *)&pba;
-	*p = *v;
-	return pba;
-}
-
-godot_variant GDAPI godot_variant_call(godot_variant *p_v, const godot_string *p_method, const godot_variant **p_args, const godot_int p_argcount, godot_variant_call_error *p_error) {
-	Variant *v = (Variant *)p_v;
+godot_node_path GDAPI godot_variant_as_node_path(const godot_variant *p_self) {
+	godot_node_path raw_dest;
+	const Variant *self = (const Variant *)p_self;
+	NodePath *dest = (NodePath *)&raw_dest;
+	*dest = *self;
```

Comment:
You're dereferencing dest and assigning something to it, that triggers the `operator =`, but the raw_dest is uninitialised, so it's undefined behaviour